src/client/pages/Checkout/CheckoutDetails/CheckoutDetails.tsx

```diff
@@ -106,6 +118,7 @@ export const CheckoutDetails = () => {
         buttonText={textKeys.CHECKOUT_FOOTER_CONTINUE_TO_PAYMENT()}
         buttonLinkTo={paymentPageLink}
         onClick={handleOnClick}
+        isLoading={createQuoteBundleMutation[1].loading}
```

Comment:
Why is this only for `[1]`? Would it make sense to have `createQuoteBundleMutation.some(m => m.loading)` to get all queries that are loading?

Reply:
Oh, now I understand - it's because you didn't destructure `createQuoteBundleMutation`. Couldn't that be done just for clarity's sake?
`const [, isLoading] = createQuoteBundleMutation`?

Reply:
yeap, much better this way